drivers/hmis/app/models/hmis/auth_policies/ce_referral_policy.rb

```diff
@@ -39,7 +39,13 @@ def can_view?
   def can_view_summary?
     return false unless Hmis::Ce.configuration.enabled?
 
-    source_project_permissions.include?(:can_manage_outgoing_referrals)
+    # Users who can manage outgoing referrals from the source project
+    return true if source_project_permissions.include?(:can_manage_outgoing_referrals)
+
+    # Users who can view the target enrollment. Bakes in the assumption that the target enrollment is in the referral's project.
+    return true if referral.target_enrollment_id.present? && project_permissions.include?(:can_view_enrollment_details)
```

Comment:
If we had an Enrollment Policy, it would be nice if we could check against that here. Maybe we can add a TODO referencing https://github.com/open-path/Green-River/issues/8549 for that.

Full Enrollment visibility currently requires both can_view_enrollment_details and can_view_project so we should check that both of those are included in project_permissions to be comprehensive. (This is the thing that would be ideal to encapsulate in the policy!)